elixir-lang/install/mint-leaf.yml

```diff
@@ -1,7 +1,7 @@
-name: mint/install-elixir
-version: 1.0.3
+name: elixir-lang/install
+version: 1.0.4
```

Comment:
I ended up bumping versions all up one minor version as well, so if we want to backfill we can guarantee it's a perfectly identical copy on all of the old versions (including the current one)